semgrep/tests/e2e/snapshots/test_check/test_timeout_threshold/error_2.txt

```diff
@@ -5,10 +5,8 @@ rules:
 - rules.forcetimeout2
 - rules.forcetimeout3
 No .semgrepignore found. Using default .semgrepignore rules. See the docs for the list of default ignores: https://semgrep.dev/docs/cli-usage/#ignoring-files
-Warning: 2 timeout error(s) in targets/equivalence/open_redirect.py when running the following rules: [forcetimeout,
-forcetimeout2]
-Semgrep stopped running rules on targets/equivalence/open_redirect.py after 2 timeout error(s). See `--timeout-
```

Comment:
Do we still keep old behavior of not running a rule after it times out 2 times?

Reply:
Hmmm I think timeouts now crash all semgrep. We might need to change that